src/lib/task.js

```diff
@@ -816,7 +817,13 @@ export class Task extends EventEmitter {
       // Build the list of container links... and base environment variables
       linkInfo = await this.states.link(this);
       // Hooks prior to running the task.
-      await this.states.created(this);
+      await promiseRetry(retry => {
+        return this.states.created(this).catch(retry);
```

Comment:
Do all of our state handlers implement a created() method that can be retried?  we should audit our features to ensure that this won't introduce some other problems.